.changes/next-release/bugfix-879edd7b-a60c-49ca-93e0-97d5bf23f2e6.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix to reduce getCodeScan API calls by increasing Intervals and latest file scan kills old running file scans."
```

Comment:
Should we remove the changelog? It's not really a bugfix from the user's perspective

Reply:
Makes sense to not add it since it is not a customer facing change. Will remove the changelog if the IDEs team reviewers feel the same.